cloudbuild/scorecard.yaml

```diff
@@ -0,0 +1,37 @@
+# Copyright 2021 Security Scorecard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+build:
+  images:
+  - gcr.io/openssf/scorecard:latest
```

Comment:
Do we use this image anywhere?

Reply:
This is the image published for others to pull down and run scorecard.

Reply:
https://github.com/ossf/scorecard#docker

Reply:
"latest" seems a bit ambiguous here, since all other tags refer to git tags. The convention seems to be to use "latest" to denote the latest release rather than main/HEAD. e.g. for https://hub.docker.com/_/python.

Should we either:
- Make "latest" point to the latest tag?
- Or rename "latest" to something else, like "main" or "git" ?

Reply:
This is the default from `cloud build`. I used the `gcloud alpha builds triggers export my-trigger  --destination=trigger.yaml` to export. 

we also need `latest` for the default docker run to work.

Reply:
Would still prefer for "latest" to point to the latest stable tag, rather than main, but we can do this later once we have a stable release process.